dev-aws/kafka-shared-msk/payment-platform/payment.tf

```diff
@@ -0,0 +1,42 @@
+resource "kafka_topic" "payment_v1_events" {
```

Comment:
we need to remove this resource from `open-banking.tf` to not cause duplicate resource name error as far as I know cc @sbuliarca

Reply:
though I assume we leave them in the module under consume/produce_topics?